dateutil/tz/tz.py

```diff
@@ -1460,6 +1458,50 @@ def __get_gettz():
         tzlocal_classes += (tzwinlocal,)
 
     class GettzFunc(object):
+        """gettz(name=None)
```

Comment:
The function prototype will be auto-generated by sphinx and should not be included.